rc/tools/make.kak

```diff
@@ -25,7 +33,7 @@ define-command -params .. \
 }}
 
 add-highlighter shared/make group
-add-highlighter shared/make/ regex "^((?:\w:)?[^:\n]+):(\d+):(?:(\d+):)?\h+(?:((?:fatal )?error)|(warning)|(note)|(required from(?: here)?))?.*?$" 1:cyan 2:green 3:green 4:red 5:yellow 6:blue 7:yellow
+add-highlighter shared/make/ regex "^%opt{make_error_highlight_pattern}?\h+(?:((?:fatal )?error)|(warning)|(note)|(required from(?: here)?))?.*?$" 1:cyan 2:green 3:green 4:red 5:yellow 6:blue 7:yellow
```

Comment:
This will not work, you'd need to have a hook that changes this highlighter whenever that option changes, and it will be made trickier as that option might have different values in different windows which means it cannot really be shared. `dynregex` might help but I have been wanting to get rid of it for a while so I would not encourage using it. Not sure what the solution would be here

Reply:
Roger that o7. I'll remove this change and leave it as it was.